src/modules/accounting/accounting.py

```diff
@@ -179,7 +179,16 @@ def _get_last_withdrawal_request_to_finalize(self, blockstamp: ReferenceBlockSta
     @lru_cache(maxsize=1)
     def _get_finalization_shares_rate(self, blockstamp: ReferenceBlockStamp) -> int:
         simulation = self.get_rebase_after_report(blockstamp)
-        shares_rate = simulation.post_total_pooled_ether * SHARE_RATE_PRECISION_E27 // simulation.post_total_shares
+        cover_shares, non_cover_shares = self.w3.lido_contracts.burner.functions.getSharesRequestedToBurn().call(
```

Comment:
You should pass _sharesRequestedToBurn on the ref slot to handleOracleReport
(https://github.com/lidofinance/lido-oracle/pull/240#discussion_r1129047356)

Reply:
+ handleOraclereport returns now array
https://github.com/lidofinance/lido-oracle/pull/240#discussion_r1129062958